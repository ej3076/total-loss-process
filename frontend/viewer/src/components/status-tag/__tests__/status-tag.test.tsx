import React from 'react';
import { create } from 'react-test-renderer';

import StatusTag from '../';

describe('<StatusTag />', () => {
  it('should render appropriately', () => {
    expect(create(<StatusTag status={1} />)).toMatchSnapshot();
    expect(create(<StatusTag status={2} />)).toMatchSnapshot();
    expect(create(<StatusTag status={3} />)).toMatchSnapshot();
    expect(create(<StatusTag status={4} />)).toMatchSnapshot();
    expect(create(<StatusTag status={23453} />)).toMatchSnapshot();
    expect(create(<StatusTag status={0} />)).toMatchSnapshot();
    expect(create(<StatusTag status={-123} />)).toMatchSnapshot();
  });
});
