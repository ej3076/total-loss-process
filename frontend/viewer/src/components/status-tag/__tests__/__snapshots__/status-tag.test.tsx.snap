// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StatusTag /> should render appropriately 1`] = `
<span
  className="bp3-tag bp3-intent-primary bp3-large bp3-minimal"
>
  <span
    className="bp3-text-overflow-ellipsis bp3-fill"
  >
    1
  </span>
</span>
`;

exports[`<StatusTag /> should render appropriately 2`] = `
<span
  className="bp3-tag bp3-intent-warning bp3-large bp3-minimal"
>
  <span
    className="bp3-text-overflow-ellipsis bp3-fill"
  >
    2
  </span>
</span>
`;

exports[`<StatusTag /> should render appropriately 3`] = `
<span
  className="bp3-tag bp3-intent-success bp3-large bp3-minimal"
>
  <span
    className="bp3-text-overflow-ellipsis bp3-fill"
  >
    3
  </span>
</span>
`;

exports[`<StatusTag /> should render appropriately 4`] = `
<span
  className="bp3-tag bp3-large bp3-minimal"
>
  <span
    className="bp3-text-overflow-ellipsis bp3-fill"
  >
    4
  </span>
</span>
`;

exports[`<StatusTag /> should render appropriately 5`] = `
<span
  className="bp3-tag bp3-large bp3-minimal"
>
  <span
    className="bp3-text-overflow-ellipsis bp3-fill"
  >
    23453
  </span>
</span>
`;

exports[`<StatusTag /> should render appropriately 6`] = `
<span
  className="bp3-tag bp3-large bp3-minimal"
>
  <span
    className="bp3-text-overflow-ellipsis bp3-fill"
  >
    0
  </span>
</span>
`;

exports[`<StatusTag /> should render appropriately 7`] = `
<span
  className="bp3-tag bp3-large bp3-minimal"
>
  <span
    className="bp3-text-overflow-ellipsis bp3-fill"
  >
    -123
  </span>
</span>
`;
